source/unified-test-format/unified-test-format.rst

```diff
@@ -1798,6 +1885,92 @@ Use a ``listIndexes`` command to check whether the index exists. Note that it is
 currently not possible to run ``listIndexes`` from within a transaction.
 
 
+loop
+~~~~
+
+The ``loop`` operation executes sub-operations in a loop until a termination
+signal is received by the unified test runner. It supports the following
+arguments:
+
+- ``operations``: the sub-operations to run on each loop iteration.
+  Each sub-operation must be a valid operation as described in this
+  specification.
+
+- ``storeErrorsAsEntity``: if specified, the runner MUST handle errors
+  arising during sub-operation execution and append a document with error
+  information to the array stored in the specified entity. If
+  ``storeFailuresAsEntity`` is specified, the runner MUST NOT include
+  failures in the errors. If ``storeFailuresAsEntity`` is not specified,
```

Comment:
I added language referencing this allowance and permitting the test runner to use the same reporting categories as it would in the driver's test suite.

Reply:
It is possible for the test scenario to route both errors and failures to the same entity, if the distinction is immaterial. However for drivers that do maintain the distinction I think it is worthwhile to keep errors and failures separate.